go/libraries/events/events.go

```diff
@@ -76,3 +98,26 @@ func (evt *Event) close() *eventsapi.ClientEvent {
 
 	return ce
 }
+
+// SetStandardAttributes sets local remote urls, and current branch name to an evt
+func SetStandardAttributes(evt *Event, dEnv *env.DoltEnv) {
+	if evt != nil {
+		if len(evt.GetAttribute(eventsapi.AttributeID_LOCAL_REMOTE_URLS)) < 1 {
+			remotes, _ := dEnv.GetRemotes()
+
+			if len(remotes) > 0 {
+				urls := make([]string, len(remotes))
+
+				for _, remote := range remotes {
+					urls = append(urls, remote.Url)
+				}
+
+				evt.SetAttribute(eventsapi.AttributeID_LOCAL_REMOTE_URLS, strings.Join(urls[:], " "))
+			}
+		}
+
+		if len(evt.GetAttribute(eventsapi.AttributeID_BRANCH_NAME)) < 1 {
+			evt.SetAttribute(eventsapi.AttributeID_BRANCH_NAME, dEnv.RepoState.Head.Ref.GetPath())
+		}
+	}
+}
```

Comment:
I'm not sure this is the right file for this code? Anyway, I figure with every command, we want to the current branch and the remote url added to the `evt.attributes` map. 

However, I wasn't sure what my strategy should be for storing a remote in the `evt.attributes` map. There may be many remotes, there may or may not be an 'origin' remote, and I'm not sure if we want the get the information for all local remotes. That said, I need to define the key for this in the `.proto` and store the value as a string. So for now, I'm just making a larger string and storing it under `LOCAL_REMOTE_URLS`. Not sure this is the ideal solution either...